src/tests/ftest/util/AgentUtils.py

```diff
@@ -27,6 +27,13 @@
 import subprocess
 import json
 import signal
+import getpass
+import socket
+import errno
+import fcntl
+import re
+
+from __future__ import print_function
```

Comment:
(pylint-misplaced-future) __future__ import is not the first non docstring statement